integration-tests/tests/workspace-dashboard.js

```diff
@@ -157,7 +157,15 @@ const setAzureAjaxMockValues = async (testPage, namespace, name, workspaceDescri
       },
       {
         metadata: {
-          resourceType: 'AZURE_STORAGE_CONTAINER'
+          resourceType: 'AZURE_STORAGE_CONTAINER',
```

Comment:
Without the fix, this causes the test to fail because we don't find the storage container we expect on the workspace dashboard page.